powerauth-admin/src/main/java/io/getlime/security/app/admin/controller/ApplicationController.java

```diff
@@ -171,7 +174,7 @@ public String applicationCreateCallback(@PathVariable("applicationId") String id
      * @param model Model with passed parameters.
      * @return "callbackUpdate" view.
      */
-    @PostMapping("/application/detail/{applicationId}/callback/update")
+    @GetMapping("/application/detail/{applicationId}/callback/update")
```

Comment:
Why are we changing a `POST` method to `GET`? In REST, `GET` endpoints should not be used to modify data. Please clarify.

Reply:
Ok, I would like to hear the reason too. But my opinion is, that the JavaDoc says
> Show application callback update form

So the the path is misleading from the very beginning.

Reply:
If you take a look at the `ApplicationController`, there are few `GET` methods, whose API path suggest data modification, however those are only used to show forms in the browser. The other `POST` methods actually request changes on PowerAuth Server using Client. Their API path usually ends with `/do.submit`.

As @banterCZ said, it is misleading naming. Before this change it was not possible to show invalid input message on the callback update form, as the redirect `return "redirect:/application/detail/" + applicationId + "/callback/create";` expects the `GET` method on that endpoint.

Reply:
Ok, let's change the naming then.

Reply:
Ok, I renamed the API paths to get update or create callback form. But misleading naming is used on many more endpoints in other controllers. I would not include it in this issue.